fastlane/lib/fastlane/documentation/actions_list.rb

```diff
@@ -44,100 +42,119 @@ def self.print_all(platform: nil)
 
     def self.show_details(filter: nil)
       puts "Loading documentation for #{filter}:".green
-
       puts ""
 
-      all_actions do |action, name|
-        next unless name == filter.strip
+      action = find_action_named(filter)
 
-        rows = []
-        rows << [action.description] if action.description
-        rows << [' ']
-        if action.details
-          rows << [action.details]
-          rows << [' ']
+      if action
+        unless action < Action
+          UI.user_error! action_subclass_error(filter)
```

Comment:
Should this really be `user_error!` so it "crashes"? On line 25 we just show the error message